components/server/src/jobs/org-only-migration-job.ts

```diff
@@ -1,57 +0,0 @@
-/**
```

Comment:
What is the lower bound of dedicated instances that we can't update from if we remove this job now? Does that affect any of Team SIDs plans regarding updating old instances (I think they want to re-create anyway, just want to make sure at this point).

Reply:
I'm pretty sure we don't have cells that old that we were creating users as billing cost centers